packages/expo-background-fetch/src/BackgroundFetch.ts

```diff
@@ -1,52 +1,64 @@
 import { Platform, NativeModulesProxy } from 'expo-core';
 import { TaskManager } from 'expo-task-manager';
 
-const { ExpoBackgroundFetch: BackgroundFetch } = NativeModulesProxy;
+const { ExpoBackgroundFetch } = NativeModulesProxy;
 
-export enum Result {
+enum Result {
   NoData = 1,
   NewData = 2,
   Failed = 3,
 }
 
-export enum Status {
+enum Status {
   Denied = 1,
   Restricted = 2,
   Available = 3,
 }
 
-export async function getStatusAsync(): Promise<Status | null> {
+async function getStatusAsync(): Promise<Status | void> {
```

Comment:
This should stay as `Promise<Status | null>` since the method sometimes returns a value.